### ISSUE TITLE
Plaid oAuth gets stuck on "Almost done..." screen

### ISSUE THREAD

**Issue by proverbface** [plaid/react-native-plaid-link-sdk#374]
# The problem

When using Plaid oAuth it's possible to get stuck on "Almost done..." screen indefinitely.

# Environment

|                        |  |
| ------------------------| ---------- |
| Plaid Link React Native          | 7.0.7                |
| ReactNative Version     | 0.63.3               |
| Occurs on Android             | yes              |
| Android OS Version                    |  any version      |
| Android Devices/Emulators    | any device                |
| Occurs on iOS     | yes            |
| iOS Version           | any version             |
| iOS Devices/Simulators    | any device                |

# Steps to Reproduce

1. Open Plaid SDK with token
2. Search and select Platypus OAuth Bank
3. Select Platypus Oauth Bank
4. Decline or simulate error
5. Once back to the app, press on Link with account numbers
6. Press on back arrow on that screen
7. Search and select Platypus OAuth Bank
8. Get stack on "Almost done..." screen

Events logged by Plaid SDK:
```
{"linkEvent": {"eventName": "OPEN", "metadata": {"errorCode": "", "errorMessage": "", "errorType": "", "exitStatus": "", "institutionId": "", "institutionName": "", "institutionSearchQuery": "", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "", "timestamp": "2021-05-04T05:12:07.140Z", "viewName": "CONSENT"}}}
{"linkEvent": {"eventName": "TRANSITION_VIEW", "metadata": {"errorCode": "", "errorMessage": "", "errorType": "", "exitStatus": "", "institutionId": "", "institutionName": "", "institutionSearchQuery": "", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "", "timestamp": "2021-05-04T05:12:08.502Z", "viewName": "SELECT_INSTITUTION"}}}
{"linkEvent": {"eventName": "SEARCH_INSTITUTION", "metadata": {"errorCode": "", "errorMessage": "", "errorType": "", "exitStatus": "", "institutionId": "", "institutionName": "", "institutionSearchQuery": "platypus", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "", "timestamp": "2021-05-04T05:12:13.226Z", "viewName": ""}}}
 {"linkEvent": {"eventName": "SELECT_BRAND", "metadata": {"errorCode": "", "errorMessage": "", "errorType": "", "exitStatus": "", "institutionId": "", "institutionName": "", "institutionSearchQuery": "", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "", "timestamp": "2021-05-04T05:12:24.503Z", "viewName": ""}}}
 {"linkEvent": {"eventName": "SUBMIT_CREDENTIALS", "metadata": {"errorCode": "", "errorMessage": "", "errorType": "", "exitStatus": "", "institutionId": "ins_127287", "institutionName": "Platypus OAuth Bank", "institutionSearchQuery": "", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "", "timestamp": "2021-05-04T05:12:38.396Z", "viewName": ""}}}
{"linkEvent": {"eventName": "OPEN_OAUTH", "metadata": {"errorCode": "", "errorMessage": "", "errorType": "", "exitStatus": "", "institutionId": "ins_127287", "institutionName": "Platypus OAuth Bank", "institutionSearchQuery": "", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "ZH1sWGPHQWSGIEG", "timestamp": "2021-05-04T05:12:42.619Z", "viewName": ""}}}
{"linkEvent": {"eventName": "OPEN", "metadata": {"errorCode": "", "errorMessage": "", "errorType": "", "exitStatus": "", "institutionId": "ins_127287", "institutionName": "Platypus OAuth Bank", "institutionSearchQuery": "", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "", "timestamp": "2021-05-04T05:12:52.202Z", "viewName": ""}}}
{"linkEvent": {"eventName": "ERROR", "metadata": {"errorCode": "INSUFFICIENT_CREDENTIALS", "errorMessage": "insufficient authorization was provided to complete the request", "errorType": "ITEM_ERROR", "exitStatus": "", "institutionId": "ins_127287", "institutionName": "Platypus OAuth Bank", "institutionSearchQuery": "", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "DI1sIoSPnprVrDO", "timestamp": "2021-05-04T05:12:53.495Z", "viewName": ""}}}
{"linkEvent": {"eventName": "TRANSITION_VIEW", "metadata": {"errorCode": "", "errorMessage": "", "errorType": "", "exitStatus": "", "institutionId": "", "institutionName": "", "institutionSearchQuery": "", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "DI1sIoSPnprVrDO", "timestamp": "2021-05-04T05:13:15.067Z", "viewName": "SELECT_INSTITUTION"}}}
{"linkEvent": {"eventName": "SEARCH_INSTITUTION", "metadata": {"errorCode": "", "errorMessage": "", "errorType": "", "exitStatus": "", "institutionId": "", "institutionName": "", "institutionSearchQuery": "platypus", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "DI1sIoSPnprVrDO", "timestamp": "2021-05-04T05:13:18.776Z", "viewName": ""}}}
{"linkEvent": {"eventName": "SELECT_BRAND", "metadata": {"errorCode": "", "errorMessage": "", "errorType": "", "exitStatus": "", "institutionId": "", "institutionName": "", "institutionSearchQuery": "", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "DI1sIoSPnprVrDO", "timestamp": "2021-05-04T05:13:20.058Z", "viewName": ""}}}
{"linkEvent": {"eventName": "SELECT_INSTITUTION", "metadata": {"errorCode": "", "errorMessage": "", "errorType": "", "exitStatus": "", "institutionId": "ins_127287", "institutionName": "Platypus OAuth Bank", "institutionSearchQuery": "", "linkSessionId": "8c255f26-4c3d-482b-a6e5-9b0d995dec09", "metadataJson": "", "mfaType": "", "requestId": "DI1sIoSPnprVrDO", "timestamp": "2021-05-04T05:13:20.975Z", "viewName": ""}}}
```

As you can see above it gets stuck on `SELECT_INSTITUTION` and does not go again through `SUBMIT_CREDENTIALS` and `OPEN_OAUTH`

# Expected Result
It should open the browser again so that the user can Grant, Decline, Simulate error again

# Screenshots
![Screen Recording 2021-05-04 at 5 16 48 PM](https://user-images.githubusercontent.com/3595076/116963373-cdd5a980-acfc-11eb-97d2-fbcffcf07646.gif)

# Logs
```
2021-05-04 17:20:57.913 518-2695/? I/ActivityTaskManager: START u0 {flg=0x4000000 cmp=XXX/com.plaid.link.LinkActivity} from uid 10165
2021-05-04 17:20:58.001 518-3888/? I/ActivityTaskManager: START u0 {flg=0x6000000 cmp=XXX/com.plaid.internal.link.LinkWebViewActivity} from uid 10165
2021-05-04 17:20:58.231 518-544/? I/ActivityTaskManager: Displayed cXXX/com.plaid.internal.link.LinkWebViewActivity: +317ms
2021-05-04 17:20:58.287 4615-4673/cXXX I/WM-WorkerWrapper: Worker result SUCCESS for Work [ id=b6775ba7-5704-439d-991d-ba6a0a3f5118, tags={ com.plaid.internal.core.analytics.batch.AnalyticsBatchUploadWorker } ]
2021-05-04 17:21:00.667 4615-4673/cXXX I/WM-WorkerWrapper: Worker result SUCCESS for Work [ id=cbe4b9dd-3506-45fb-8b19-85d78bf85617, tags={ com.plaid.internal.core.analytics.batch.AnalyticsBatchUploadWorker } ]
2021-05-04 17:21:03.447 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to font at 'https://cdn.plaid.com/link/2.0.1003/assets/fonts/cern-regular.woff2' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a (0)
2021-05-04 17:21:03.480 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to font at 'https://cdn.plaid.com/link/2.0.1003/assets/fonts/cern-medium.woff2' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a (0)
2021-05-04 17:21:03.596 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to font at 'https://cdn.plaid.com/link/2.0.1003/assets/fonts/cern-semibold.woff2' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a (0)
2021-05-04 17:21:03.614 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to font at 'https://cdn.plaid.com/link/2.0.1003/assets/fonts/cern-extrabold.woff2' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a (0)
2021-05-04 17:21:03.614 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to font at 'https://cdn.plaid.com/link/2.0.1003/assets/fonts/micr-e13b-webfont.woff2' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a (0)
2021-05-04 17:21:09.638 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:09.725 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:09.840 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:09.931 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:10.616 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:18.871 518-631/? I/ActivityTaskManager: START u0 {act=android.intent.action.VIEW dat=https://cdn.plaid.com/... cmp=com.android.chrome/com.google.android.apps.chrome.IntentDispatcher} from uid 10165
2021-05-04 17:21:19.065 518-631/? I/ActivityTaskManager: START u0 {act=android.intent.action.VIEW dat=https://cdn.plaid.com/... flg=0x14002000 cmp=com.android.chrome/org.chromium.chrome.browser.ChromeTabbedActivity (has extras)} from uid 10111
2021-05-04 17:21:27.964 518-4516/? I/ActivityTaskManager: START u0 {act=android.intent.action.VIEW cat=[android.intent.category.BROWSABLE] dat=plaid://redirect/?oauth_state_id=d8b48414-d53e-4e3d-8ede-658e63f48f52 flg=0x14000000 pkg=cXXX cmp=cXXX/com.plaid.internal.LinkRedirectActivity (has extras)} from uid 10111
2021-05-04 17:21:28.011 518-4516/? I/ActivityTaskManager: START u0 {flg=0x24000000 cmp=cXXX/com.plaid.internal.link.LinkWebViewActivity (has extras)} from uid 10165
2021-05-04 17:21:28.017 518-4516/? W/ActivityTaskManager: Duplicate finish request for r=ActivityRecord{26d2b19 u0 cXXX/com.plaid.internal.LinkRedirectActivity t468 f}}
2021-05-04 17:21:28.046 4615-4615/cXXX W/Heap: Could not find BuildConfig
    com.heapanalytics.android.internal.HeapException: Heap was not configured properly. Check logcat for Heap build warnings and resolution steps. (Build Config resource ID = 0.)
        at com.heapanalytics.android.internal.BuildConfigRetrieverImpl.getBuildConfig(BuildConfigRetrieverImpl.java:15)
        at com.heapanalytics.android.internal.AppState.getBuildConfigField(AppState.java:302)
        at com.heapanalytics.android.internal.AppState.getApplicationInfo(AppState.java:209)
        at com.heapanalytics.android.internal.ApplicationMetadataManager.isPersistedDifferent(ApplicationMetadataManager.java:35)
        at com.heapanalytics.android.internal.HeapApplicationLifecycleTracker.maybeSendAppLifecycleEvent(HeapApplicationLifecycleTracker.java:38)
        at com.heapanalytics.android.internal.HeapApplicationLifecycleTracker.receiveAppForeground(HeapApplicationLifecycleTracker.java:29)
        at com.heapanalytics.android.internal.HeapSessionTrackerImpl.appForegrounded(HeapSessionTrackerImpl.java:33)
        at com.heapanalytics.android.internal.HeapActivityLifecycleCallbacks.unsafeOnActivityStarted(HeapActivityLifecycleCallbacks.java:155)
        at com.heapanalytics.android.internal.HeapActivityLifecycleCallbacks.onActivityStarted(HeapActivityLifecycleCallbacks.java:127)
        at android.app.Application.dispatchActivityStarted(Application.java:406)
        at android.app.Activity.dispatchActivityStarted(Activity.java:1348)
        at android.app.Activity.onStart(Activity.java:1833)
        at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:467)
        at com.plaid.internal.go0.onStart(SourceFile:1)
        at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1435)
        at android.app.Activity.performStart(Activity.java:8024)
        at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3475)
        at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
        at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
        at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:183)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:132)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
2021-05-04 17:21:28.047 4615-4615/cXXX W/Heap: Could not find BuildConfig
    com.heapanalytics.android.internal.HeapException: Heap was not configured properly. Check logcat for Heap build warnings and resolution steps. (Build Config resource ID = 0.)
        at com.heapanalytics.android.internal.BuildConfigRetrieverImpl.getBuildConfig(BuildConfigRetrieverImpl.java:15)
        at com.heapanalytics.android.internal.AppState.getBuildConfigField(AppState.java:302)
        at com.heapanalytics.android.internal.AppState.getApplicationInfo(AppState.java:209)
        at com.heapanalytics.android.internal.ApplicationMetadataManager.persistCurrentApplicationInfo(ApplicationMetadataManager.java:27)
        at com.heapanalytics.android.internal.HeapApplicationLifecycleTracker.receiveAppForeground(HeapApplicationLifecycleTracker.java:34)
        at com.heapanalytics.android.internal.HeapSessionTrackerImpl.appForegrounded(HeapSessionTrackerImpl.java:33)
        at com.heapanalytics.android.internal.HeapActivityLifecycleCallbacks.unsafeOnActivityStarted(HeapActivityLifecycleCallbacks.java:155)
        at com.heapanalytics.android.internal.HeapActivityLifecycleCallbacks.onActivityStarted(HeapActivityLifecycleCallbacks.java:127)
        at android.app.Application.dispatchActivityStarted(Application.java:406)
        at android.app.Activity.dispatchActivityStarted(Activity.java:1348)
        at android.app.Activity.onStart(Activity.java:1833)
        at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:467)
        at com.plaid.internal.go0.onStart(SourceFile:1)
        at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1435)
        at android.app.Activity.performStart(Activity.java:8024)
        at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3475)
        at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
        at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
        at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:183)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:132)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
2021-05-04 17:21:28.052 4615-5064/cXXX D/AppsFlyer_6.2.0: Calling onAppOpenAttribution with:
    {install_time=2021-03-24 03:10:02, path=/, scheme=plaid, link=plaid://redirect/?oauth_state_id=d8b48414-d53e-4e3d-8ede-658e63f48f52, host=redirect, oauth_state_id=d8b48414-d53e-4e3d-8ede-658e63f48f52}
2021-05-04 17:21:28.227 518-544/? D/ArtManagerInternalImpl: /data/misc/iorapd/cXXX/1000079/com.plaid.internal.link.LinkWebViewActivity/compiled_traces/compiled_trace.pb doesn't exist
2021-05-04 17:21:28.228 518-544/? I/ActivityTaskManager: Displayed cXXX/com.plaid.internal.link.LinkWebViewActivity: +150ms
2021-05-04 17:21:28.296 4615-5373/cXXX I/AppsFlyer_6.2.0: data: {"country":"US","af_timestamp":"1620105688141","appsflyerKey":"9********************5","isFirstCall":"false","customData":"{\"brazeCustomerId\":\"1a0bdb16-c90e-429f-9d95-7bb70fbe1d0f\"}","registeredUninstall":false,"operator":"T-Mobile","af_deeplink":"plaid:\/\/redirect\/?oauth_state_id=d8b48414-d53e-4e3d-8ede-658e63f48f52","isGaidWithGps":"true","app_version_code":"1000079","model":"sdk_gphone_x86","brand":"google","deviceType":"user","deviceData":{"cpu_abi":"x86","build_display_id":"RSR1.200819.001.A1","btch":"no","dim":{"size":"2","ydp":"420.0","xdp":"420.0","y_px":"1794","x_px":"1080","d_dpi":"420"},"arch":"","btl":"100.0","cpu_abi2":""},"exception_number":0,"disk":"4966\/5951","sdk":"30","app_version_name":"2.0.13","device":"generic_x86_arm","cksm_v1":"023aa392b218c522df3e4d638cc96fa2ca","af_events_api":"1","prev_session_dur":42,"cell":{"mnc":260,"mcc":310},"network":"MOBILE","timepassedsincelastlaunch":"51","open_referrer":"android-app:\/\/cXXX","af_v2":"80504146f650c1fe6c80d1b3f7c21e254443a532","sig":"FAC61745DC0903786FB9EDE62A962B399F7348F0BB6F899B8332667591033B9C","uid":"1616555421360-2833316888394458680","lang_code":"en","installDate":"2021-03-24_031002+0000","firstLaunchDate":"2021-03-24_031021+0000","ivc":false,"lang":"English","last_boot_time":1620105554694,"product":"sdk_gphone_x86","date2":"2021-05-04_124235+1200","counter":"142","date1":"2021-03-24_161002+1300","advertiserId":"f8f239b8-451a-4f8f-8b95-414b1e3595de","advertiserIdEnabled":"true","af_v":"c8aac97ed8b006261048542738fe484d4e99d25d","carrier":"Android","af_preinstalled":"false","iaecounter":"1616","sc_o":"p","tokenRefreshConfigured":false,"appUserId":"juan.proverbio+119@dovetailstudios.com","kef656b":"4dea65bd72e19e8848191f1b0c531b0d591a120d5a1f13","platformextension":"android_reactNative"}
2021-05-04 17:21:28.999 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to link element resource at 'https://cdn.plaid.com/experiments/optimizely/datafile' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a&receivedRedirectUri=plaid%3A%2F%2Fredirect%2F%3Foauth_state_id%3Dd8b48414-d53e-4e3d-8ede-658e63f48f52 (0)
2021-05-04 17:21:29.349 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to font at 'https://cdn.plaid.com/link/2.0.1003/assets/fonts/cern-medium.woff2' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a&receivedRedirectUri=plaid%3A%2F%2Fredirect%2F%3Foauth_state_id%3Dd8b48414-d53e-4e3d-8ede-658e63f48f52 (0)
2021-05-04 17:21:29.365 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to font at 'https://cdn.plaid.com/link/2.0.1003/assets/fonts/cern-regular.woff2' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a&receivedRedirectUri=plaid%3A%2F%2Fredirect%2F%3Foauth_state_id%3Dd8b48414-d53e-4e3d-8ede-658e63f48f52 (0)
2021-05-04 17:21:29.365 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to font at 'https://cdn.plaid.com/link/2.0.1003/assets/fonts/cern-semibold.woff2' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a&receivedRedirectUri=plaid%3A%2F%2Fredirect%2F%3Foauth_state_id%3Dd8b48414-d53e-4e3d-8ede-658e63f48f52 (0)
2021-05-04 17:21:29.365 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to font at 'https://cdn.plaid.com/link/2.0.1003/assets/fonts/cern-extrabold.woff2' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a&receivedRedirectUri=plaid%3A%2F%2Fredirect%2F%3Foauth_state_id%3Dd8b48414-d53e-4e3d-8ede-658e63f48f52 (0)
2021-05-04 17:21:29.365 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to font at 'https://cdn.plaid.com/link/2.0.1003/assets/fonts/micr-e13b-webfont.woff2' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a&receivedRedirectUri=plaid%3A%2F%2Fredirect%2F%3Foauth_state_id%3Dd8b48414-d53e-4e3d-8ede-658e63f48f52 (0)
2021-05-04 17:21:35.897 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:35.975 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:36.067 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:36.100 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:36.183 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to XMLHttpRequest at 'https://sandbox.plaid.com/link/brands/search' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a&receivedRedirectUri=plaid%3A%2F%2Fredirect%2F%3Foauth_state_id%3Dd8b48414-d53e-4e3d-8ede-658e63f48f52 (0)
2021-05-04 17:21:36.217 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:36.250 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:36.331 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to XMLHttpRequest at 'https://sandbox.plaid.com/link/brands/search' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a&receivedRedirectUri=plaid%3A%2F%2Fredirect%2F%3Foauth_state_id%3Dd8b48414-d53e-4e3d-8ede-658e63f48f52 (0)
2021-05-04 17:21:36.367 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:36.650 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:36.665 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to XMLHttpRequest at 'https://sandbox.plaid.com/link/brands/search' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a&receivedRedirectUri=plaid%3A%2F%2Fredirect%2F%3Foauth_state_id%3Dd8b48414-d53e-4e3d-8ede-658e63f48f52 (0)
2021-05-04 17:21:36.765 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:36.798 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:36.882 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to XMLHttpRequest at 'https://sandbox.plaid.com/link/brands/search' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a&receivedRedirectUri=plaid%3A%2F%2Fredirect%2F%3Foauth_state_id%3Dd8b48414-d53e-4e3d-8ede-658e63f48f52 (0)
2021-05-04 17:21:37.099 4615-4615/cXXX I/chromium: [INFO:CONSOLE(0)] "Access to XMLHttpRequest at 'https://sandbox.plaid.com/link/brands/search' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.6&reactNativeLinkSdkVersion=7.0.2&androidSdkImplementation=inProcess&token=link-sandbox-b292e754-21d1-47d2-92fc-012edb33d35a&receivedRedirectUri=plaid%3A%2F%2Fredirect%2F%3Foauth_state_id%3Dd8b48414-d53e-4e3d-8ede-658e63f48f52 (0)
2021-05-04 17:21:44.649 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:47.744 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change
2021-05-04 17:21:48.149 518-594/? D/InputDispatcher: Waiting to send key to Window{b942d54 u0 cXXX/com.plaid.internal.link.LinkWebViewActivity} because there are unprocessed events that may cause focus to change

```

**Comment by AlexD10S** [comment]
Did you find something, I am experiencing the same issue

**Comment by iarmankhan** [comment]
is this solved yet? Experiencing the same on android

**Comment by proverbface** [comment]
This issue is still occurring on iOS and Android on  "react-native-plaid-link-sdk": "7.0.7"

**Comment by bjjeong** [comment]
@proverbface did you ever resolve this issue?

**Comment by proverbface** [comment]
@bjjeong I haven't tried the latest SDK, I don't believe is fixed yet. But it's such an important thing to be fixed. There is no workaround as far as I know.

**Comment by amytang0** [comment]
Hey sorry we missed this ticket, I couldn't find the session you posted earlier; would you mind sharing `link_session_id`'s from more recently? Thanks!

**Comment by dtroupe-plaid** [comment]
Closing because Plaid Link React Native 7.x.x is no longer supported. If the issue persists in the latest SDK please open a new issue.
